### DISCUSSION TITLE
메세지폼 디자인 수정이 필요합니다. (임시)

### DISCUSSION THREAD

**Discussion by GiPyoo** [ecnvjs/About-view#31] [General]
@spiderq2697 

지긍 스파이더님이 해주신 디자인에서느,ㄴ

이런이란 향태의 데이터를 처리하기에 너무 복잡한니다/.

아래에 있는 메뉴들을 상단의 그룹형태로 바꾸는 건 어떨까요?

**Comment by SingTheCode** [comment]
좋슘다

**Reply by GiPyoo** [reply]
@Team-Cavascript/about
